.api/v1.5.0/schema.pbiviz.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
I think that we should remove the ```.api``` directory in the repository and install the api automatically. Please take a look at [one](https://github.com/Microsoft/PowerBI-visuals-ForceGraph/blob/master/package.json#L13) and [two](https://github.com/Microsoft/PowerBI-visuals-ForceGraph/blob/master/package.json#L49) Do you think it's reasonable?